ui/src/app/utils/stores.tsx

```diff
@@ -112,6 +113,18 @@ export const profileStore = atom<ProfileStore>({
     }),
 });
 
+export interface CreateNewUserSatisfactionSurveyStore {
```

Comment:
I implemented this as a store instead of component state so users don't lose their responses if they accidentally escape the modal (by clicking out of it, pressing escape, or clicking "Cancel").